src/components/internal/DataViewFilters.jsx

```diff
@@ -24,49 +25,88 @@ const StyledDiv = styled.div`
 	}
 `;
 
-const PopoverDiv = styled.div`
-	&.loading {
-		opacity: .5;
-		pointer-events: none;
-	}
-`
-
 function DataViewFilters(props) {
 	const { t } = useMosaicTranslation();
+	const [state, setState] = useState({
+		anchorEl : null,
+		dropdownOpen : false
+	});
 
-	const activeFilters = props.activeFilters || [];
-	
+	const activeFilters = props.activeFilters.value || [];	
 	const primaryFilters = props.filters.filter(val => val.type === "primary");
 	const primaryFilterNames = primaryFilters.map(val => val.name);
 	const optionalFilters = props.filters.filter(val => val.type !== "primary");
 	
 	const active = optionalFilters.filter(val => activeFilters.includes(val.name));
 	const options = optionalFilters.map(val => ({ label : val.label, value : val.name }));

```

Comment:
Everything seems correct, the only thing I noticed after reading the ticket again is that it asks for the options to be sorted alphabetically, so I think we could do a .sort() after mapping, that way it's always sorted and we don't have to worry about sorting every time there's a search